app/src/main/java/com/zackhsi/kiva/activities/LoanReviewActivity.java

```diff
@@ -113,6 +114,7 @@ protected void onActivityResult (int requestCode, int resultCode, Intent data) {
                     String accountId = new KivaProxy().getKivaProxyId();
                     payment.setUserId(accountId);
                     payment.saveEventually();
+                    Toast.makeText(getApplicationContext(), loan.name + " says 'Thanks!'", Toast.LENGTH_LONG).show();
```

Comment:
This is fine because it'll be called when the activity exists. The one I fixed was making a toast in a API callback after the fragment was no longer attached to it's activity.